### PR TITLE
React Native 0.63 example

### PR DESCRIPTION
**Pull request by twometresteve**
Adds a new example for React Native 0.63 replacing the example for 0.59, which is no longer supported.

Github has possibly been more helpful than needed and detected that a number of files have been moved and changed, despite me having performed the delete separately and after adding the new files.

I have tested the example on both Android 5 and iOS 10 devices, noting that React Native 0.63 itself does not support iOS 9.3 (our current earliest support iOS version).

### CODE REVIEW

File: examples/reactnative/rn_0.59/README.md [hunk 1]
```diff
@@ -1,46 +0,0 @@
-# React Native
```
Comment (fractalwrench) [inline examples/reactnative/rn_0.59/README.md:1]: It'd be worth copying this over to the 0.63 project and adapting where necessary

File: examples/reactnative/rn063example/android/app/build.gradle [hunk 2]
```diff
@@ -73,11 +78,10 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    entryFile: "index.js"
+    enableHermes: false,  // clean and rebuild if changing
 ]
 
 apply from: "../../node_modules/react-native/react.gradle"
-apply from: "../../node_modules/@bugsnag/react-native/bugsnag-react-native.gradle"
```
Comment (fractalwrench) [inline examples/reactnative/rn063example/android/app/build.gradle:80]: this is required to resolve the AAR dependency on Android

File: examples/reactnative/rn063example/android/app/src/main/java/com/rn063example/CrashyModule.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.reactnative;
+package com.rn063example;
 
 import com.facebook.react.bridge.ReactApplicationContext;
```
Comment (twometresteve) [inline examples/reactnative/rn063example/android/app/src/main/java/com/rn063example/CrashyModule.java:3]: handledError() is missing from this class.

File: examples/reactnative/rn_0.59/README.md (hunk 1 above)
Comment (twometresteve) [reply examples/reactnative/rn_0.59/README.md:1]: Absolutely.  I actually meant to and forgot.

File: examples/reactnative/rn063example/android/app/build.gradle (hunk 2 above)
Comment (twometresteve) [reply examples/reactnative/rn063example/android/app/build.gradle]: It seemed to work fine without, but more than happy to add - especially if it's something that can fail intermittently.

Comment (twometresteve) [reply examples/reactnative/rn063example/android/app/build.gradle]: Ok, it's coming in the next release.  I've added this and bumped the dependency to the next alpha release.  CI will fail initially, but once the alpha is created it should pass on retry.

File: examples/reactnative/rn063example/android/app/src/main/java/com/rn063example/CrashyModule.java (hunk 3 above)
Comment (twometresteve) [reply examples/reactnative/rn063example/android/app/src/main/java/com/rn063example/CrashyModule.java:4]: Now added.

**Pull request merged**